ruma-common/src/push/predefined.rs

```diff
@@ -8,6 +8,21 @@ use super::{
 
 use ruma_identifiers::UserId;
 
+macro_rules! set {
```

Comment:
We're already using the `maplit` crate in other Ruma crates, might as well add a dependency here too and use `btreeset` from there.